storage/tools/common.go

```diff
@@ -39,12 +43,25 @@ func GetStorageFromFlagsOrDie(treeId trillian.LogID) storage.LogStorage {
 			panic(err)
 		}
 
-		return store
+		return store, err
 	}
 
-	panic(fmt.Sprintf("Unknown storage type: %s", *storageTypeFlag))
+	return nil, fmt.Errorf("Unknown storage type: %s", *storageTypeFlag)
 }
 
-func GetLogServingPort() int {
-	return *logPortFlag
-}
\ No newline at end of file
+// GetStorageFromFlags returns a configured storage instance, errors are fatal and if func init() {
+// returns the storage can be used.
+func GetStorageFromFlagsOrDie(treeId trillian.LogID) storage.LogStorage {
+	storage, err := GetStorageFromFlags(treeId)
+
+	if err != nil {
+		panic(err)
+	}
+
+	return storage
+}
+
+// Gets the port number to be used when serving log data
```

Comment:
Done. Funny how Go ignores the existing conventions of every other language and then insists you follow theirs exactly. Grumble, mutter etc.